### PR TITLE
JBERET-532 jberet-support is not update with jberet.core 2.x

### PR DESCRIPTION
**Pull request by Zerrossetto**
https://issues.redhat.com/projects/JBERET/issues/JBERET-532

### CODE REVIEW

File: src/main/java/org/jberet/support/io/CassandraBatchlet.java [hunk 1]
```diff
@@ -34,7 +35,7 @@ public class CassandraBatchlet extends CassandraReaderWriterBase implements Batc
     /**
      * {@inheritDoc}
      * <p>
-     * This method executes the cql statement(s) as specified in {@link #cql} batch property,
+     * This method executes the cql state	ment(s) as specified in {@link #cql} batch property,
```
Comment (chengfang) [inline src/main/java/org/jberet/support/io/CassandraBatchlet.java:38]: need to remove extra spaces here

File: pom.xml [hunk 2]
```diff
@@ -280,21 +284,15 @@
...
-            <scope>test</scope>
-        </dependency>
+		<dependency>
+		    <groupId>jakarta.annotation</groupId>
+		    <artifactId>jakarta.annotation-api</artifactId>
+		</dependency>
+		<dependency>
+		    <groupId>jakarta.transaction</groupId>
+		    <artifactId>jakarta.transaction-api</artifactId>
+		</dependency>
```
Comment (chengfang) [inline pom.xml:295]: in a few places there are extra tabs in front of deps element. Pls reformat.

**Comment by Zerrossetto** [comment]
@chengfang now all tests passed. I'm not so sure everything's fine on git side because I'm still unexperienced on the collaborative side of things.
Please, have a look

**Review by chengfang — APPROVED** [review]

**Comment by chengfang** [comment]
@Zerrossetto thanks!

**Pull request merged**